code-challenges.test.js

```diff
@@ -15,6 +19,34 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+// a describe method that lists the name of the function OR naming of the particular test.
+
```

Comment:
Great setup. Please install jest and run the test suites. Then place the results of the test suite for the test and then the function. After you comment out the helper code that your text editor VScode added, you should be able to.